schema.sql

```diff
@@ -4,8 +4,9 @@ CREATE TABLE animals (
     id INT GENERATED ALWAYS AS IDENTITY,
 	name VARCHAR(15) NOT NULL,
 	date_of_birth DATE NOT NULL,
-    escape_attempts INT NOT NULL, 
+  escape_attempts INT NOT NULL, 
 	neutered BOOLEAN NOT NULL,
 	weight_kg DECIMAL NOT NULL,
+	species VARCHAR(15),
```

Comment:
- [ ] Please use `ALTER TABLE` to add new column `species` to the animals table.
**Take a Look**
```sql
ALTER TABLE animals ADD species varchar(255);
```